pkg/assembler/backends/arangodb/backend.go

```diff
@@ -303,10 +317,18 @@ func GetBackend(ctx context.Context, args backends.BackendArgs) (backends.Backen
 		// repeat this for the collections where an edge is going into
 		hasSBOMEdges.To = []string{hasSBOMsStr}
 
+		var certifyVulnEdges driver.EdgeDefinition
+		certifyVulnEdges.Collection = certifyVulnEdgesStr
+		// define a set of collections where an edge is going out...
+		certifyVulnEdges.From = []string{pkgVersionsStr, certifyVulnsStr}
+
+		// repeat this for the collections where an edge is going into
+		certifyVulnEdges.To = []string{certifyVulnsStr, cvesStr, ghsasStr, osvsStr}
```

Comment:
hmm, how would we handle the `NoVuln` case here, thoughts?

Reply:
oh good call. I missed `noVuln`. Let me update to account for that.

Reply:
Based on the discussion, `noVuln` will be added as a field on `certifyVuln`. That is not currently implemented by this PR (only adds CVE, GHSA and OSV). Another PR will capture `certifyVuln` along with `noVuln` field within it.